core/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java

```diff
@@ -112,7 +112,7 @@ private void copySubmissionFilesToReport(JPlagResult result) {
                 if (relativeFilePath.isEmpty()) {
                     relativeFilePath = file.getName();
```

Comment:
We could also just preface the files name with a `/`. Would be shorter, simpler solution. 
What do we prefer?

Reply:
To be honest, I have no real preference. But related to pre-release panic, I say fewer changes are better! :D

Reply:
No, I would say it is fine as it is right now.